cmd/app/add.go

```diff
@@ -72,6 +73,7 @@ func NewAddCommand(clients *shared.ClientFactory) *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&addFlags.orgGrantWorkspaceID, cmdutil.OrgGrantWorkspaceFlag, "", cmdutil.OrgGrantWorkspaceDescription())
+	cmd.Flags().StringVarP(&addFlags.environmentFlag, "environment", "E", "", "environment of app (local, deployed)")
```

Comment:
note: The `slack install` command now supports the `--environment <local|deployed>` values. This flag can be used to by-pass the prompt.